cmds/ps/ps.go

```diff
@@ -168,17 +174,61 @@ func (pT *ProcessTable) PrepareString() {
 	pT.fstring = fstring
 }
 
+func mapSubset(check map[byte]int, ref map[byte]int) bool {
+	for cLet, cNum := range check {
+		if rNum, ok := ref[cLet]; ok {
+			if cNum != rNum {
+				return false
+			}
+		} else {
+			return false
+		}
+	}
+	return true
+}
+
+func isPermutation(check string, ref string) bool {
```

Comment:
You could also use `map[rune]uint` to actually count letters if you want a real anagram check.